Deep_learning/Deep-learning.rst

```diff
@@ -270,230 +270,196 @@ PyTorch
 Building PyTorch for ROCm
 **************************
 
-This is a quick guide to setup PyTorch with ROCm support inside a docker container. Assumes a .deb based system. See `ROCm install <https://github.com/RadeonOpenCompute/ROCm#supported-operating-systems---new-operating-systems-available>`_ for supported operating systems and general information on the ROCm software stack.
 
-Note: Currently, ROCm install version 3.3 is required.
+Option 1 (Recommended) : Use Docker image with PyTorch Pre-installed
+********************************************************************
 
-1. Install or update rocm-dev on the host system:
+Using Docker gives you portability and access to a pre-built docker container that has been rigorously tested within AMD. This might also save on the compilation time and should perform exactly as it did when tested, without facing potential installation issues.
 
-::
+1. Pull the latest public PyTorch Docker image 
 
-  sudo apt-get install rocm-dev
-  or
-  sudo apt-get update
-  sudo apt-get upgrade
+docker pull rocm/pytorch:latest
```

Comment:
script mode using backticks (left of 1 key on keyboard)
`docker pull rocm/pytorch:latest`